src/components/TogetherButton/TogetherButton.js

```diff
@@ -1,12 +1,43 @@
 import React from 'react';
 import PropTypes from 'prop-types';
-import styles from './TogetherButton.module.css';
+import './TogetherButton.css';
 
-const TogetherButton = () => (
-  <div className={styles.TogetherButton} data-testid="TogetherButton">
-    TogetherButton Component
-  </div>
-);
+class TogetherButton extends React.Component {
+  constructor(props){
+    super(props);
+    this.state={
+      togetherON: false,
+      text: 'Start Collaborating'
+    }
+}
+
+ToggleButton(){
+  if (this.state.togetherON) {
+    this.setState((currentState) => ({
+      togetherON: !currentState.togetherON,
+    }));
+    this.state.text = 'Start Collaborating';
+  } else {
+    this.setState((currentState) => ({
+      togetherON: !currentState.togetherON,
+    }));
+    this.state.text = 'Done Collaborating';
+  }
+
+  window.TogetherJS(this);
+  return false;
+}
+
+render(){
```

Comment:
will do it now

Reply:
okay after looking at them, feeling a little bit confused. Could you give some suggestions? I thought it was because the indentation was 2 spaces not a tab but I saw in Map it's also 2 spaces. (Also a react convention just googled)

Reply:
Oh I didn't mean the number of spaces, I mean the whole code block needs to be shifted 1 tab since this is under the TogetherButton component? (Correct me if I'm wrong)